src/App.php

```diff
@@ -787,6 +790,12 @@ public static function reset(): void
         self::$shutdown = [];
         self::$options = [];
         self::$sorted = false;
+        self::$routes = [
+            self::REQUEST_METHOD_GET       => [],
+            self::REQUEST_METHOD_POST      => [],
+            self::REQUEST_METHOD_PUT       => [],
+            self::REQUEST_METHOD_PATCH     => [],
+            self::REQUEST_METHOD_DELETE    => [],
+        ];
```

Comment:
This was missing from `reset()`. Was that intentional or a mistake?

Reply:
Sure, but if I want to reset App, shouldn't it reset the routes? Then, the routes should be rebuilt after. It seems odd that the docblock says:

> Reset all the static variables

And shutdown, init, errors, are all reset, but routes isn't.